src/angular/styles/includes/_mixins.scss

```diff
@@ -164,6 +164,53 @@
   }
 }
 
+@mixin horizontalBorderAndShadowAsBoxShadow($attached: below, $borderWidth: 2, $scalingFactor: 1) {
+  $sign: 1;
+  @if $attached == above {
+    $sign: -1;
+  }
+  box-shadow: boxShadow($sign, $borderWidth, $scalingFactor),
+    horizontalBorderBoxShadow(-$sign, $borderWidth, $scalingFactor);
+}
+
+@function boxShadow($sign, $borderWidth, $scalingFactor) {
+  @return 0 pxToRem($sign * $borderWidth * 4 * $scalingFactor) 0 rgba(0, 0, 0, 0.15);
+}
+
+@function horizontalBorderBoxShadow($sign, $borderWidth, $scalingFactor) {
+  @return 0 pxToRem($sign * ($borderWidth + 1) * $scalingFactor) 0
+    pxToRem(-$borderWidth * $scalingFactor) $sbbColorCloud;
+}
+
+@mixin panelOffsetCorrection($attached: below, $borderWidth: 2, $scalingFactor: 1) {
+  $sign: -1;
+  @if $attached == above {
+    $sign: 1;
+  }
+  $offset-top: $sign * ($borderWidth - 1) * $scalingFactor;
+
+  @if $offset-top < 0 {
+    $offset-top: $offset-top - 0.05; // better rendering e.g. in 4k mode
+  }
+  @if $offset-top > 0 {
+    $offset-top: ceil($offset-top);
+  }
+  position: relative;
+  top: pxToRem($offset-top);
+}
+
+@mixin borderLeftAsAfter($color) {
```

Comment:
Is there a specific reason as to why use ::after instead of ::before?

Reply:
no, changed it to before